ci-operator/step-registry/rosa/aws/sts/conformance/test/rosa-aws-sts-conformance-test-commands.sh

```
@@ -0,0 +1,35 @@
+fi
+
+set -x
+openshift-tests run "${TEST_SUITE}" ${TEST_ARGS:-} \
+    --provider "${TEST_PROVIDER}" \
+    -o "${ARTIFACT_DIR}/e2e.log" \
+    --junit-dir "${ARTIFACT_DIR}/junit" 2>&1 | tee /tmp/openshift-tests.log
+
+exit_code=${PIPESTATUS[0]}
+set +x
+
+if [[ -n "${SKIP_MONITOR_TEST:-}" ]] && [[ $(grep "failed due to a MonitorTest failure" /tmp/openshift-tests.log) ]]; then
```

Shellcheck (0.11.0)

[style] Use grep -q instead of comparing output with [ -n .. ].

(SC2143)